src/VisualStudio/Core/Def/Implementation/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -91,18 +92,13 @@ private enum GlobalNotificationState
                 try
                 {
                     // make sure connection is done right
-                    _ = await client._rpc.InvokeWithCancellationAsync<string>(
-                        nameof(IRemoteHostService.Connect), new object[] { clientId, uiCultureLCID, cultureLCID, TelemetryService.DefaultSession.SerializeSettings() }, cancellationToken).ConfigureAwait(false);
+                    _ = await client._endPoint.InvokeAsync<string>(
+                        nameof(IRemoteHostService.Connect),
+                        new object[] { clientId, uiCultureLCID, cultureLCID, TelemetryService.DefaultSession.SerializeSettings() },
+                        cancellationToken).ConfigureAwait(false);
 
                     success = true;
                 }
-                catch (Exception e)
-                {
-                    // translate to our own cancellation if it is raised.
-                    cancellationToken.ThrowIfCancellationRequested();
-
-                    throw SoftCrash(workspace, hubClient, e, "Failed to connect to ServiceHub", cancellationToken);
-                }
```

Comment:
why did this go away?

Reply:
All remote call related exceptions are handled lower - in the methods called by this method.